cuML/CMakeLists.txt

```diff
@@ -71,6 +70,7 @@ set(CMAKE_IGNORE_PATH "${CMAKE_INSTALL_DIR}/lib" CACHE STRING
 set(GPU_ARCHS "" CACHE STRING
   "List of GPU architectures (semicolon-separated) to be compiled for")
 option(LINEINFO "Enable lineinfo in nvcc" OFF)
+option(MEMCHECK "Enable cudamemcheck function symbol backtrace in nvcc" OFF)
```

Comment:
If you're using this to enable symbol names in backtrace, it's probably better to rename this to `BACKTRACE` or something more appropriate?